dbt/include/duckdb/macros/materializations/external.sql

```diff
@@ -0,0 +1,91 @@
+{% materialization external, adapter="duckdb", supported_languages=['sql', 'python'] %}
+
+  {%- set location = config.require('location') -%}
+  {%- set format = config.get('format', default='parquet') -%}
+  {%- set delimiter = config.get('delimiter', default=';') -%}
+  -- set language - python or sql
+  {%- set language = model['language'] -%}
+
+  {%- set target_relation = this.incorporate(type='view') %}
+  -- check if the location exists
+  {%- set location_exists = location_exists(location) %}
```

Comment:
This can be:

`{%- set location_exists = adapter.location_exists(location) %}`

unless you had a reason for adding in extra macro as a layer of indirection?

Reply:
We need to get rid of the database name, so that is why